ui/src/api/types.ts

```diff
@@ -162,6 +162,7 @@ export interface Symbol extends BaseEntity {
 
 export interface SymbolAttributes extends BaseEntityAttributes {
   tex: string | null;
+  type: "identifier" | "function" | "operator";
```

Comment:
will this ever get extended?  if so, can  you make a note here what other code need to get changed?

Reply:
Sure thing! Good thinking 👍

Reply:
@kyleclo I thought it was more in the spirit of your suggestion to add the note about extensions to `data-processing/common/types.py`, where one is likely to add new names of types of symbols. Does my change to the documentation there satisfy your desire for documentation about extensions?

Reply:
yea that's fine